target_chains/ethereum/contracts/forge-test/Pyth.Aave.t.sol

```diff
@@ -134,4 +146,127 @@ contract PythAaveTest is Test, WormholeTestUtils, PythTestUtils, RandTestUtils {
         vm.expectRevert(PythErrors.PriceFeedNotFound.selector);
         pythOracleGetter.getAssetPrice(addr);
     }
+
+    function testGetAssetPriceRevertsIfPriceIsNegative() public {
+        PriceFeedMessage[] memory priceFeedMessages = new PriceFeedMessage[](1);
+        PriceFeedMessage memory priceFeedMessage = PriceFeedMessage({
+            priceId: getRandBytes32(),
+            price: int64(-5), // assuming price should always be positive
+            conf: getRandUint64(),
+            expo: int32(getRandInt8() % 13), // pyth contract guarantees that expo between [-12, 12]
```

Comment:
you don't need this assumption for this contract, so i don't think you should make it. Use the assumptions mandated by the implementation instead.